casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -66,6 +66,8 @@ class MultiParentCasperMergeSpec extends FlatSpec with Matchers with Inspectors
     checkMerge("Nil", echoContract(1), echoContract(2))
   }
 
+  // TODO: Peek rows/column
+  // Note this skips pairs that lead to infinite loops
```

Comment:
we should include tests for those ultimately, as the merge result for them is clear: they're mregable. So this should be another TOOO :)

Reply:
How? Infinite loops are going to run out of phlo and any changes they make on the TS are going to be reverted. I see no point in testing such cases

Reply:
Having such a test costs near to 0, and prevents us from unexpected results - like a block conflicting despite there being a way to merge it.

Notice that running out of phlo doesn't mean the deploy won't make it to the network. It will, because we need to charge for it. We might not replay it (or we may) - but those are all issues separate from what the result of 'is this a conflict' is.

Finally, I've been wrong in my `result is clear: they're mergable` statement: `CX !!X` leads to an infinite loop, yet must be a conflict.

Hope this convinces you that it's worth adding those cases?